src/app/public/modules/vertical-tabset/vertical-tabset-group.component.ts

```diff
@@ -63,12 +63,23 @@ import {
 })
 export class SkyVerticalTabsetGroupComponent implements OnInit, OnDestroy {
 
+  /**
+   * Indicates whether to disable the ability to expand and collapse the group.
+   * @default "false"
+   */
   @Input()
   public disabled: boolean;
 
+  /**
+   * Specifies the header for the collapsible group of tabs.
+   * @default "false"
+   */
```

Comment:
I think you mean for the `@default` annotation to go on the `open` property?

Reply:
Nice catch. Thanks! I've updated the defaults to be on the 2 boolean properties and not the `groupHeading` property.